desktop-fonts/symbols-nerd-font/autobuild/build

```diff
@@ -1,2 +1,10 @@
 abinfo "Installing Symbols Nerd Font ..."
 install -Dvm644 -t "$PKGDIR"/usr/share/fonts/TTF "$SRCDIR"/*.ttf
+install -Dvm644 \
```

Comment:
This is really another step, please add an abinfo here indicating that you are installing the fontconfig.